### PR TITLE
PNDA-4012: Top level component type added

### PR DESCRIPTION
**Pull request by shubhi0310**
# Problem Statement:
- PNDA-4012 : Missing application type for higher level items

# Change:
- Display the application type for higher level items in application summary popup on applications page.

### CODE REVIEW

File: console-frontend/partials/modals/application-status.html [hunk 1]
```diff
@@ -40,25 +40,72 @@ <h4>{{name}} summary</h4>
                 <div class="panel-body">
                     <div class="collapse" id="oozieSubComponents">
                     <div ng-if="oozieComponent.actions != undefined" class="row margin-left-20">
-                            <div id="workflows" ng-repeat="workflow in oozieComponent.actions" 
-                            class="margin-left-20">
-                             <div class="row">
+
```
Comment (willis27) [inline console-frontend/partials/modals/application-status.html]: remove empty row

File: console-frontend/partials/modals/application-status.html [hunk 2]
```diff
@@ -40,25 +40,72 @@ <h4>{{name}} summary</h4>
...
-                            <div class="col-md-2 margin-left-neg-35">-</div>
+                            <div class="col-md-2 margin-left-neg-35"  ng-if="!workflow.applicationType">-</div>
+                            <div class="col-md-2 margin-left-neg-35" ng-if="workflow.applicationType">{{workflow.applicationType}}</div>
                             <div class="col-md-2 margin-left-3">
-                            <span ng-attr-title="{{workflow.oozieId}}" class="word-break">{{ workflow.oozieId }}</span>
+                            <span ng-if="workflow.oozieId" class="word-break">{{ workflow.oozieId }}</span>
+                            <span ng-if="workflow.yarnId" class="word-break">
+                                <a href="{{yarnUrl +'/' + workflow.yarnId}}" target="_blank">{{ workflow.yarnId }}</a>
+                            </span>
                             </div>
```
Comment (willis27) [inline console-frontend/partials/modals/application-status.html]: remove empty row

File: console-frontend/partials/modals/application-status.html [hunk 3]
```diff
@@ -40,25 +40,72 @@ <h4>{{name}} summary</h4>
...
+                                    <tr>
+                                         <td>Failed</td>
+                                         <td>{{ workflow.information.stageSummary.failed }}</td>
+                                         <td>Failed</td>
+                                         <td>{{ workflow.information.jobSummary.failed }}</td>
+                                    </tr>
+                              </table>
+                            </div>
                             </div>
                             <div class="collapse" id="oozieWorkflow-{{$index}}">
```
Comment (willis27) [inline console-frontend/partials/modals/application-status.html]: remove empty row.

**Review by willis27 — COMMENTED** [review]
Please address code styling issues (removing those empty rows between <div> sections.

**Comment by shubhi0310** [comment]
@willis27: I have removed blank lines. Please let me know if anything else is required.

**Pull request merged**